src/SFA.DAS.Approvals/InnerApi/Requests/BulkCreateReservationsWithNonLevyRequest.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace SFA.DAS.Approvals.InnerApi.Requests
+{
+    public class BulkCreateReservationsWithNonLevyRequest
```

Comment:
Reservations are created irrespective of employer type so no need for the `NonLevy` class name